app/store/wsMiddleware.ts

```diff
@@ -53,7 +64,7 @@ const socketMiddleware = () => {
         }
         break
       default:
-        console.log('default')
+        console.log(event.type)
```

Comment:
can we add some clarity here:
```suggestion
        console.log(`received websocket event: ${event.type}`)
```